modules/xfem/test/tests/crack_tip_enrichment/tests

```diff
@@ -6,6 +6,11 @@
     map = false
     unique_id = true
     superlu = true
+    requirement = 'The XFEM module shall have the capability to perform crack '
+                  'tip enrichment in 2D.'
```

Comment:
The XFEM module shall have the capability to include near-tip enrichment functions in 2D mechanics simulations.